.ci/spellcheck/.pyspelling.wordlist.txt

```diff
@@ -550,4 +550,6 @@ YOLOv
 yolov
 Zongyuan
 ZeroScope
-zeroscope
\ No newline at end of file
+zeroscope
+librispeech
```

Comment:
please keep alphabetical order adding words in wordlist

Reply:
Done